Source Code/Helpers.cs

```diff
@@ -164,8 +164,10 @@ public static PlayerControl playerById(byte id)
             }
         }
 
-        private static List<int> lighterColors = new List<int>(){ 3, 4, 5, 7, 10, 11};
+        private static List<int> lighterColors = new List<int>(){ 3, 4, 5, 7, 10, 11 };
```

Comment:
Should we add these ids to the CustomColors.lighterColors?

Reply:
Not sure if it should go into CustomColors, as they are default colors.
However could revert changes to Helpers.cs, and use Reflection in CustomColors, to have a shared Reference to "lighterColors" between Helpers and CustomColors